src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String BUY_CONSTANT = "buy,";
+    private static final String COMMA = ",";
+    private static final String RESULT_CONSTANT = "result,";
+    private static final String SUPPLY_CONSTANT = "supply,";
+    private final Map<String, String> statisticsCache = new HashMap<>();
+
+    public void writeStatistic(String fromFileName, String newFile) {
```

Comment:
don't forget to put it after public method

Reply:
fix comment